content/docs/pipelines/sdk/python-based-visualizations.md

```diff
@@ -0,0 +1,105 @@
++++
+title = "Python Based Visualizations"
+description = "Predefined and custom visualizations of pipeline outputs"
+weight = 80
++++
+
+This page describes Python based visualizations, how to create them, and how to
+use them to visualize results within the Kubeflow Pipelines UI. Python based
+visualizations are available in Kubeflow Pipelines version
+[0.1.28](https://github.com/kubeflow/pipelines/releases/tag/0.1.28) and will
+be avaliable in Kubeflow version 0.7.0.
+
+While Python based visualizations are intended to be the main method of
+visualizing data within the Kubeflow Pipelines UI, they do not replace the
+previous method of visualizing data within the Kubeflow Pipelines UI. When
+considering which method of visualization to use within your pipeline, check the
+[limitations of python based visualizations](https://github.com/kubeflow/
```

Comment:
s/python/Python

Please check all occurrences in this doc for correct branding.